src/components/question-builder/components/add-answer.js

```diff
@@ -44,7 +42,8 @@ const AddAnswer = ({
           <Button
             className="button-secondary"
             text="Submit"
-            onClick={() => (submitHandler(), onAnswerSubmit())}
+            onClick={() => onAnswerSubmit(answerItemCreate(answerInputText))}
```

Comment:
This button is already in a form with an onSubmit handler. You can remove the click handler from here and make the button type="submit" and it will fire the onSubmit on the form element reducing path of execution to a single instance.

Reply:
Oh thats sweet, I'll make the change.